frontend/src/public/css/style.css

```diff
@@ -19,4 +19,25 @@
 .card-row{
     color: #283583; 
     margin: auto
+}
+
+body {
+    display: flex;
```

Comment:
Flex is normally used for smaller parts of a page rather than the whole body, but I suppose if this is working with the simple pages we're creating then it's not an issue.